vnext/Microsoft.ReactNative.Managed/packages.lock.json

```diff
@@ -63,7 +63,7 @@
       "NETStandard.Library": {
         "type": "Transitive",
         "resolved": "2.0.3",
-        "contentHash": "548M6mnBSJWxsIlkQHfbzoYxpiYFXZZSL00p4GHYv8PkiqFBnnT68mW5mGEsA/ch9fDO9GkPgkFQpWiXZN7mAQ==",
+        "contentHash": "st47PosZSHrjECdjeIzZQbzivYBJFv6P2nv4cj2ypdI204DO+vZ7l5raGMiX4eXMJ53RfOIg+/s4DHVZ54Nu2A==",
```

Comment:
>st47PosZSHrjECdjeIzZQbzivYBJFv6P2nv4cj2ypdI204DO [](http://example.com/codeflow?start=24&length=48)

I wonder why the content hash is different. On my machine I also have a different one. Is it machine-specific?

Reply:
It is not machine specific, but I have seen this happen frequently. It appears that some dependencies change their contents without changing the version number.